lib/client/index.js

```diff
@@ -44,7 +45,12 @@ module.exports = async ({ port = null, config = {}, filters = {} }) => {
 
   if (commitSigningEnabled(config)) {
     const commitSigningRules = commitSigningFilterRules();
-    filters.private?.push(...commitSigningRules);
+    if (config.BROKER_BOOT_MODE == 'universal') {
+      filters['github'].private?.push(...commitSigningRules);
+      filters['github-enterprise'].github.private?.push(...commitSigningRules);
+    } else {
+      filters.private?.push(...commitSigningRules);
```

Comment:
@aarlaud, since commit signing is very GitHub-centric (and checks GitHub API endpoints), should we leave to `github` and `github-enterprise` only?

Reply:
clarified